### DISCUSSION TITLE
Add programmability of tablet pen button bindings to osu options

### DISCUSSION THREAD

**Discussion by Akosih97** [ppy/osu#30750] [Ideas]
I am playing with a hovering pen and without a pen nib in order to increase my effective hovering distance on the tablet. For operating the game outside of gameplay I usually use one of the pen buttons as a mouse click. 

At the moment, I did not find a possibility of setting this in game, so I still have to start my OpenTabletDriver in order to set the pen button bindings and can then override the area options using the osu options.

**Comment by Banana090** [comment]
Adding to that it would be useful to bind more functionality to the tablet pen from inside osu since I often find myself using mouse for a lot of things while in menu. But it feels like all the interactions should be possible using only pen + keyboard

**Reply by rvnexe** [reply]
opening right-click tooltip is horrible, specially in Song Select
